README.md

```diff
@@ -1,7 +1,120 @@
-# Tour Savvy App
+# Tour Savvy App Backend
+<a name="readme-top"></a>
 
+# 📗 Table of Contents
 
-## Authors:
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Api documentation](#api-doc)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Tour_Savvy_Backend] <a name="about-project"></a>
+
+Tour_Savvy is a rails api app that allows operations like get, delete, post for packages and reservations. It also supports authentication for user login and signup.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <ul>
+    <li><a href="https://www.postgresql.org/">Postgresql v15.3</a></li>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails v7.1.1</a></li>
+    <li><a href="https://rspec.info/">Rspec v3.12</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Index and Show Packages**
+- **Index and Show Reservations**
+- **Add/Remove Reservation**
+- **Add/Remove Package**
+- **Authentication and authorization**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
```

Comment:
- [ ]  You are using Rspec and rswag, but the "test" folder still exists, which is used by another test framework. Kindly consider deleting that folder if you are not using it. 👍

Reply:
@Meltrust 
unnecessary test folder and files are removed.

Thanks for your feedback 👍 😃